apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -573,55 +567,17 @@ fn parent_based(sampler: opentelemetry::sdk::trace::Sampler) -> opentelemetry::s
 }
 
 impl Conf {
-    pub(crate) fn calculate_field_level_instrumentation_ratio(&self) -> Result<f64, Error> {
```

Comment:
I think that this PR is maybe ol, but this part needs to be reverted.

Reply:
if the field level sampling has nothing to do with the global otel sampling, then we should keep the change, no? Or is it that we could end up with FTV1 having a higher sampling than otel, so we have to go back to the code that divides the ratios?

Reply:
In the context of this PR it has nothing to do with it.
FTV1 is requested for a proportion of sampled Otel traces.It is not true that if an Otel trace is being sampled then it always has FTV1.

Reply:
ok, I see

Reply:
fixed in 54f01367